ol/cli/src/server.rs

```diff
@@ -31,7 +31,13 @@ pub async fn start_server(mut node: Node, _run_checks: bool) {
         let interval = interval(Duration::from_secs(10));
         let stream = IntervalStream::new(interval);
         let event_stream = stream.map(move |_| {
-            let vitals = Vitals::read_json(&path);
+            let vitals = match Vitals::read_json(&path) {
+                Ok(v) => v,
+                Err(e) => {
+                    println!("Error reading json cache: \nError {}", e);
+                    exit(1)
```

Comment:
I don't think we want to exit here. This would cause the web-monitor server to crash. Let's just return an error.